src/System/Integrate/Http/Karnel.php

```diff
@@ -7,16 +7,17 @@
 use System\Container\Container;
 use System\Http\Request;
 use System\Http\Response;
+use System\Router\Router;
 
 class Karnel
 {
     /** @var Container */
     protected $app;
 
-    /** @var array<int, class-string> Global middleware */
+    /** @var array<int, class-string|callable> Global middleware */
```

Comment:
we need anonymous class?

Reply:
you right, using anonymous will untracked middleware was run.